.github/actions/compile-examples/compile.sh

```diff
@@ -0,0 +1,27 @@
+set -e
+SKETCH="$1"
+BOARD="$2"
+LOG_LEVEL="$3"
+LIBS="$4"
+
+HOME=/home/blues
+IFS=','
+read -ra libraries <<< "$LIBS"
+for lib in "${libraries[@]}"; do 
+      lib=$( echo $lib | xargs )          # remove leading/trailing whitespace
+      args=""
+      export ARDUINO_LIBRARY_ENABLE_UNSAFE_INSTALL=false
+      if [[ $lib == *"/"* ]]; then
```

Comment:
I know this is why/what/how you are doing it based on a conversation or comment elsewhere, so I thought it would be good to put it where the rubber meets the road.
```suggestion
      if [[ $lib == *"/"* ]]; then        # presence of forward-slash indicates URL
```